docs/docs/troubleshooting/connection-issues.mdx

```diff
@@ -41,14 +23,9 @@ This procedure will erase all settings, such as Bluetooth profiles, output selec
 
 :::
 
-### Acquiring a Reset UF2
-
-First, acquire the reset UF2 image file with one of the following options:
+### Building a Reset UF2
```

Comment:
I've never been a big fan of the UF2 being in this header. "Building Reset Firmware" maybe?